packages/ERTP/test/swingsetTests/splitPayments/bootstrap.js

```diff
@@ -1,7 +1,11 @@
 import { E } from '@agoric/eventual-send';
 import { makeIssuerKit } from '../../../src';
 
+const { details: X } = assert;
+
 export function buildRootObject(vatPowers, vatParameters) {
+  const arg0 = vatParameters.argv[0];
+  
```

Comment:
I think linting would fix this, but there's an extra indent here

```suggestion

```

Reply:
Linting did fix it